packages/digitalproperty-app/lib/cmds/landregistry_cmds/addTitles.js

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env node
```

Comment:
Filename is `addTitles.js` but the file being used as teh yargs description for a new Person?  
File should be named something like `addPerson.js`

Avoids confusion later.